docs/rules/no-dynamic-require.md

```diff
@@ -18,7 +18,5 @@ require(name());
 ### Pass
 
 ```js
-require('../name');
```

Comment:
I thought the purpose was to illustrate that both a string literal, and a template literal with no interpolations, would pass?